java_vertx_grpc_bench/src/main/java/vertx/Main.java

```diff
@@ -14,9 +15,11 @@
 public class Main {
     public static void main(String[] args) {
         Vertx vertx = Vertx.vertx();
-        vertx.nettyEventLoopGroup().forEach(u -> {
-            vertx.deployVerticle(new ServerVerticle());
-        });
+        DeploymentOptions deploymentOptions = new DeploymentOptions();
+        int cores = Runtime.getRuntime().availableProcessors();
```

Comment:
Will this obtain a correct value of available processors? That is to say, in a Dockerized environment, will it grab the host's available parallelism or the container's? In my experience, it's usually the latter; this is the reason why the `GRPC_SERVER_CPUS` environmental variable is exposed to help the server properly determine how much is available.

Reply:
IIRC currect the later JVM will works after JDK 11. I will update.